openedxstats/settings/base.py

```diff
@@ -21,10 +21,10 @@
 
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'CHANGE THIS!!!'
+SECRET_KEY = 'nedhatestriaging'
```

Comment:
Oh now he looks at PRs


-------- Original message --------
From: Ned Batchelder <notifications@github.com>
Date: 7/12/16 5:38 PM (GMT-05:00)
To: zrrobbins/openedxstats <openedxstats@noreply.github.com>
Cc: "Robbins, Zachary R" <zrrobbins@wpi.edu>, State change <state_change@noreply.github.com>
Subject: Re: [zrrobbins/openedxstats] Chart bugfix (#14)


In openedxstats/settings/base.py<https://github.com/zrrobbins/openedxstats/pull/14#discussion_r70527313>:

> @@ -21,10 +21,10 @@
>
>
>  # SECURITY WARNING: keep the secret key used in production secret!
> -SECRET_KEY = 'CHANGE THIS!!!'
> +SECRET_KEY = 'nedhatestriaging'


cute

-
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://github.com/zrrobbins/openedxstats/pull/14/files/0c11fd53f17eeaca86192ef47d2d1e24d126beab#r70527313>, or mute the thread<https://github.com/notifications/unsubscribe/AGc9kBJEDu7jA44gPFG58VR7gQjSeet-ks5qVAkcgaJpZM4JKzif>.